.github/workflows/release_droid_release_on_maven_central.yml

```diff
@@ -0,0 +1,47 @@
+name: Release Droid - Release On Maven Central
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { scala-version: 'scala2.12', pk-skip: true }
+          - { scala-version: 'scala2.13', pk-skip: false }
```

Comment:
why does it skip pgp for 2.13?

Reply:
Skip is for project-keeper, okay I am going to rename it.

The main issue is that for dependency artifacts have `_2.12` or `_2.13` compatibility suffix. By default we use `2.13` version, but when I cross compile for other versions, these dependencies are not in changes or dependencies files (because of changed suffix). So I skip project-keeper checks for them.